tasks.py

```diff
@@ -1404,8 +1414,16 @@ def check_already_current(tag=None, sha=None):
         'compile_schema': 'Compile the schema documentation first (default: False)',
     }
 )
-def docs_server(c, address='localhost:8080', compile_schema=False):
+def docs_server(
+    c, address='localhost:8080', extract_settings=True, compile_schema=False
+):
     """Start a local mkdocs server to view the documentation."""
+    if extract_settings:
```

Comment:
Q (nb): Do we need this option? Is there a case where the settings file is not required for rendering?